homeassistant/components/vesync/sensor.py

```diff
@@ -236,7 +249,7 @@ class VeSyncSensorEntity(VeSyncBaseEntity, SensorEntity):
 
     def __init__(
         self,
-        device: VeSyncAirBypass | VeSyncOutlet | VeSyncSwitch,
+        device: VeSyncAirBypass | VeSyncOutlet | VeSyncSwitch | VeSyncBaseDevice,
```

Comment:
Isn't everything based on Basedevice?

Reply:
It used to be just `VeSyncBaseDevice` and got updated to `VeSyncAirBypass | VeSyncOutlet | VeSyncSwitch` in
https://github.com/home-assistant/core/commit/0dc1e7d1e64a0172c49a1114f811797283c12de7. That might have been done to give better function discovery.  But humidifier devices are different. I think we can just use `VeSyncBaseDevice` since lambdas are by nature generic.